packages/@lwc/synthetic-shadow/src/faux-shadow/focus.ts

```diff
@@ -104,14 +105,23 @@ const focusableTagNames = {
 // This function based on https://allyjs.io/data-tables/focusable.html
 // It won't catch everything, but should be good enough
 // There are a lot of edge cases here that we can't realistically handle
-function isFocusable(element: HTMLElement): boolean {
-    const tagName = tagNameGetter.call(element);
-    return (
+function isFocusable(element: HTMLElement, hostElement: HTMLElement): boolean {
```

Comment:
`hostElement` is probably `Element` now that we have adjusted that... same for element

Reply:
maybe not... if this is only for custom elements, then hostElement is HTMLElement

Reply:
`hostElement` is always a custom element and `element` is every element in the ancestor chain (composed) between the `target` and the `hostElement`.